src/toxcore/dht/dht_node.rs

```diff
@@ -112,17 +112,34 @@ impl DhtNode {
                 None => {
                     match self.saddr_v4 {
                         Some(v4) => Some(SocketAddr::V4(v4)),
-                        None => None,
+                        None => {
+                            warn!("to_packed_node() fails because get_socket_addr returns None");
```

Comment:
Correct message text to something like:
"get_socket_addr: failed to get address of DhtNode"